index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="container">  
+  <header>
+    <div class="top-header"><img src="images/Time.png" alt="time">
+    <img src="images/wifi.png" alt="wifi">
+  </div>
+    <div class="bottom-header">
+      <h1>Welcome</h1>
+      <span class="identical-to">&#8801;</span>
```

Comment:
- [ ] This hamburger menu should be wrapped in a `nav` tag instead of the `span` tag, this is because this will be later on developed to be clickable to show us the mobile menu. It is the correct semantic tag to use for the navbar for our project

Reply:
Good, corrected, thanks!